docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md

```diff
@@ -114,7 +114,7 @@ The most important one is support for [`URL`](https://developer.mozilla.org/en-U
 
 Another popular feature request was support for the [`.forEach` method on DOM collection](https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach). Since `core-js` already polyfilled iterators of DOM collections, why not add also `.forEach` to `NodeList` and `DOMTokenList`?
 
-#### Removed obsolete featues:
+#### Removed obsolete features:
```

Comment:
Fwiw, markdown best practice includes not including trailing `:` in headings. But that's far outside the scope of a spelling PR.